services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None,
+               actors_ids: list = None) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids,
+                                   actors__id__in=actors_ids)
```

Comment:
When both `genres_ids` and `actors_ids` are provided, the current logic applies an `AND` condition between genres and actors. If you intended to apply an `OR` condition, you should adjust the logic accordingly. Consider using `Q` objects to combine filters with `OR` if that's the desired behavior.